juniper_official/Server-Monitoring/kube-metrics/check-deployment-status-replicas.rule

```diff
@@ -0,0 +1,59 @@
+healthbot {
+    topic kube-metrics {
+        rule check-deployment-status-replicas {
+            keys [ deployment namespace ];
+            synopsis "";
+            description "Checks kube metrics deployment replica status";
```

Comment:
Checks that replica set status is not unavailable.

Reply:
Added description.